File: quick_sort.js
const quickSort = (array) => {
    if(array.length < 2) {
        return array;
    }
    const pivot = array[0];
    const left = array.slice(1).filter(ele => ele <= pivot);
    const right = array.slice(1).filter(ele => ele > pivot);
    return quickSort(left).concat([pivot], quickSort(right)); 
} 

File: quick_sort.py
def quick_sort(unsorted_list):
    if len(unsorted_list) < 2:
        return unsorted_list
    comparator = unsorted_list.pop(0)
    left = list(filter(lambda x: x < comparator, unsorted_list))
    right = list(filter(lambda x: x >= comparator, unsorted_list))
    return quick_sort(left) + [comparator] + quick_sort(right)
